src/main/java/gregtech/integration/hwyla/provider/BlockOreDataProvider.java

```diff
@@ -19,7 +21,7 @@ public class BlockOreDataProvider implements IWailaDataProvider {
 
     public void register(@NotNull IWailaRegistrar registrar) {
         registrar.registerBodyProvider(this, BlockOre.class);
-        registrar.addConfig(GTValues.MODID, "gregtech.block_ore");
+        registrar.addConfig("GregTech", "gregtech.block_ore");
```

Comment:
would it be better for these strings to be a `public static final` field in GTValues? but it's not really blocking imo